### ISSUE TITLE
Build Action fails to setup: Github doesn't recognize `working-directory`

### ISSUE THREAD

**Issue by xypnox** [withastro/action#37]
Earlier builds passed with the log: 

```
Secret source: Actions
Prepare workflow directory
Prepare all required actions
Getting action download info
Download action repository 'actions/checkout@v3' (SHA:f43a0e5ff2bd294095638e18286ca9a3d1956744)
Download action repository 'withastro/action@v1' (SHA:f122c0232093b3504e4c841a46c2eebd07c47392)
Getting action download info
Download action repository 'pnpm/action-setup@v2' (SHA:d882d12c64e032187b2edb46d3a0d003b7a43598)
Download action repository 'oven-sh/setup-bun@v1' (SHA:a1800f471a0bc25cddac36bb13e6f436ddf341d7)
Download action repository 'actions/setup-node@v3' (SHA:1a4442cacd436585916779262731d5b162bc6ec7)
Download action repository 'actions/upload-pages-artifact@v1' (SHA:84bb4cd4b733d5c320c9c9cfbc354937524f4d64)
Getting action download info
Download action repository 'actions/upload-artifact@v3' (SHA:a8a3f3ad30e3422c9c7b888a15615d19a852ae32)
Complete job name: build
```

However, a run triggered by the latest version resulted in:

```
Secret source: Actions
Prepare workflow directory
Prepare all required actions
Getting action download info
Download action repository 'actions/checkout@v3' (SHA:f43a0e5ff2bd294095638e18286ca9a3d1956744)
Download action repository 'withastro/action@v1' (SHA:64c331d10da115fbaab6c13ceb8d64f06199a703)
Error: withastro/action/v1/action.yml (Line: 96, Col: 7): Unexpected value 'working-directory'
Error: GitHub.DistributedTask.ObjectTemplating.TemplateValidationException: The template is not valid. withastro/action/v1/action.yml (Line: 96, Col: 7): Unexpected value 'working-directory'
   at GitHub.DistributedTask.ObjectTemplating.TemplateValidationErrors.Check()
   at GitHub.Runner.Worker.ActionManifestManager.ConvertRuns(IExecutionContext executionContext, TemplateContext templateContext, TemplateToken inputsToken, String fileRelativePath, MappingToken outputs)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
Error: Fail to load withastro/action/v1/action.yml
```

**Comment by xypnox** [comment]
Repository builds for reference: https://github.com/xypnox/xypnox.github.io/actions/workflows/astro.yml

**Comment by natemoo-re** [comment]
Closed in https://github.com/withastro/action/pull/36, released as [v1.0.5](https://github.com/withastro/action/releases/tag/v1.0.5).

**Comment by natemoo-re** [comment]
Okay I updated the `v1` tag to include the changes in `v1.0.5` which fixes it, but now the deployment via `actions/deploy-pages@v2` isn't working. I'm going to rollback the changes so I can figure this out.

**Comment by xypnox** [comment]
In the meantime I have replaced `uses: withastro/action@v1` with `uses: withastro/action@v1.0.3` and I can build my project. I will update to latest once this gets fixed and can help test the new versions.

**Comment by natemoo-re** [comment]
I was able to track down the other issue that was breaking things, release going out shortly!

**Comment by natemoo-re** [comment]
Confirmed that this particular issue is fixed in [`v1.0.6`](https://github.com/withastro/action/releases/tag/v1.0.6) and the `v1` tag has been updated! Please open a new issue if you run into other problems.

**Comment by xypnox** [comment]
Can confirm the new version works, I have reverted to `uses: withastro/action@v1`.

Thanks!
